packages/quantic/force-app/main/default/lwc/quanticTabBar/e2e/quanticTabBar.e2e.ts

```diff
@@ -0,0 +1,199 @@
+    primaryExpectation: 'the first two tabs',
+    secondaryExpectation: '',
+    numberOfTabsDisplayed: 2,
+    numberOfOptionsInDropdown: 2,
+    moreButtonLabel: 'More',
+  },
+];
+
+useCaseTestCases.forEach((useCase) => {
+  let test = fixtures[useCase.value];
+  test.describe(`quantic tab bar ${useCase.label}`, () => {
+    test.describe("when the container's width can fit all the tabs", () => {
+      test('should display all the tabs without displaying the dropdown list', async ({
+        tabBar,
+      }) => {
+        const expectedNumberOfTabs = 4;
+        const tabs = tabBar.allVisibleTabs;
+
+        expect(tabs).not.toBeNull();
+        expect(await tabs.count()).toEqual(expectedNumberOfTabs);
+
+        const isMoreDropdownVisible = await tabBar.dropdown.isVisible();
+        expect(isMoreDropdownVisible).toBe(false);
+      });
+    });
+
+    viewportWidths.forEach(
+      ({
+        label,
+        width,
+        primaryExpectation,
+        secondaryExpectation,
+        numberOfTabsDisplayed,
+        numberOfOptionsInDropdown,
+        moreButtonLabel,
+      }) => {
+        test.describe(`when the viewport is resized to a ${label} width`, () => {
+          test.use({
+            viewport: {width: width, height: standardViewportHeight},
+          });
+          test(`should display only ${primaryExpectation}, the other tabs should be displayed in the dropdown list ${secondaryExpectation}`, async ({
+            tabBar,
+          }) => {
+            const displayedTabs = tabBar.allVisibleTabs;
+
+            expect(displayedTabs).not.toBeNull();
+            expect(await displayedTabs.count()).toEqual(numberOfTabsDisplayed);
+
+            const isMoreDropdownVisible = await tabBar.dropdown.isVisible();
+            expect(isMoreDropdownVisible).toBe(true);
+
+            await tabBar.clickMoreButton();
+
+            const dropdownOptions = tabBar.allDropdownOptions;
+            expect(dropdownOptions).not.toBeNull();
+            expect(await dropdownOptions.count()).toEqual(
+              numberOfOptionsInDropdown
+            );
+
+            const dropdownLabel = await tabBar.moreButtonLabel.textContent();
+            expect(dropdownLabel).toEqual(moreButtonLabel);
+          });
+        });
+      }
+    );
+
+    test.describe('when the more button is clicked', () => {
+      // Using medium width (300px), 2 tabs should be displayed and 2 should be in the dropdown.
+      test.use({
+        viewport: {width: mediumViewportWidth, height: standardViewportHeight},
+      });
+      test('should open the dropdown and display the options', async ({
+        tabBar,
+      }) => {
+        const expectedNumberOfDropdownOptions = 2;
+        await tabBar.clickMoreButton();
+
+        const dropdownOptions = tabBar.allDropdownOptions;
+        expect(dropdownOptions).not.toBeNull();
+        expect(await dropdownOptions.count()).toEqual(
+          expectedNumberOfDropdownOptions
+        );
+      });
```

Comment:
You already tested that no? You already tested a medium width in the previous loop